GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigSettingsPage.Designer.cs

```diff
@@ -90,15 +89,14 @@ private void InitializeComponent()
             this.Global_FilesEncoding.Location = new System.Drawing.Point(129, 413);
             this.Global_FilesEncoding.Name = "Global_FilesEncoding";
             this.Global_FilesEncoding.Size = new System.Drawing.Size(241, 21);
-            this.Global_FilesEncoding.TabIndex = 17;
             // 
             // btnCommitTemplateBrowse
             // 
+            this.btnCommitTemplateBrowse.AccessibleName = "Browse Path to commit template";
```

Comment:
These strings should probably be translated.
The translation app is not picking this up right now (this is the first use of AccessibleName in this project).
Short term, translation strings could be created separately (but how hard can it be to fix the translationapp, to make sure the build fails if not translated?)

Reply:
AccessibleName (and AccessibleDescription) added to the translation app in #8713.
@guybark any comments on the items to translate?
#8713 should be merged soon, then this PR should be rebased on master and translation run once more.

Reply:
That's a good point, thanks for raising it. The AccessibleName properties needs to be as translated as visual text labels are. (So by default, translated, except for things like branding.) The AccessibleName is what gets announced by screen reader for customers who are blind or have low vision, so needs to be in a language that the customer wants. I hard-coded the strings because in the same file there seemed to be hard-coded Text properties, and I assumed the app wasn't localized. It sounds like I made the wrong assumption. If the AccessibleName properties won't be translated, the change should be canceled until it's possible to have the strings translated.

Reply:
Regarding the removal of the use of TabIndex, WinForms apps are vulnerable to both the tab order and the programmatic order of the UI not being a good match for the use of the UI. The tab order is relatively straightforward to fix in VS through use of its "Tab Order" feature (in the View menu) but that doesn't fix the programmatic order. The only way to fix the programmatic order is to ensure that the calls to Add() in the designer.cs file match the logical order of the UI. The existing TabIndex use overrides that, so if the TabIndex use is removed the Add() order will dictate the tab order and programmatic order. If the form is later opened up in the VS designer, the TabIndex use will be added back but in an order that matches the Add() order, so the tab order and programmatic order remain good. A point was raised above that things may break later when controls are added or removed, and that's right. All WinForms apps are vulnerable, and the only way for things to work well for customers is for app builders to include the maintenance of the tab order and programmatic order in the list of things they do as their app evolves over time.

Reply:
AccessibleName will be translated when you rebase, the TranslationApp will pick up this field and add it to the source English.xlf, translated on Transifex.
https://github.com/gitextensions/gitextensions/wiki/How-To%3A-build-instructions#update-english-translations

So for order: The easiest way around seem to be to just make sure that there is no TabIndex